modules/taskbar.less

```diff
@@ -4,7 +4,7 @@
 	display: none;
 	z-index: 10001;
 	width: 50%;
-	left: 50%;
```

Comment:
Seemed weird that it had 50% width to begin with, so now it's back to 100%

On Tue, 15 Sep 2015 11:35 James Game <notifications@github.com> wrote:

> In modules/taskbar.less
> <https://github.com/NodeBB/nodebb-theme-persona/pull/108#discussion_r39525553>
> :
>
> > @@ -4,7 +4,7 @@
> >  	display: none;
> >  	z-index: 10001;
> >  	width: 50%;
> > -	left: 50%;
>
> For some reason when I was testing it, the Chrome inspector showed left
> being set to 0.
> As I noticed it originally was set to 50%, but later overridden by
> something else, I wanted to make sure that it's forced to be 50%
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NodeBB/nodebb-theme-persona/pull/108/files#r39525553>.
>